desktop/models/user.coffee

```diff
@@ -59,8 +60,16 @@ module.exports = class User extends Backbone.Model
     CurrentUser = require './current_user.coffee'
     LoggedOutUser = require './logged_out_user.coffee'
 
-    CurrentUser.orNull() or
-    new LoggedOutUser attributes
+    user = CurrentUser.orNull() or new LoggedOutUser attributes
+
+    if user instanceof CurrentUser
+      RavenClient.setUserContext
```

Comment:
Not relevant for this model, but something to be aware of is model code is meant to be used universally and therefore it's best to avoid dependencies beyond data and "business logic". Ideally, a client-side only library like this would be moved outside the model and into global client-side setup code. This would be a blocker if it were the `CurrentUser` model for instance because that model wraps the session on the server and client.